round4/team-05.md

```diff
@@ -0,0 +1,162 @@
+2. Simple and intuitive interface for users
+3. Customizable filters for each user
+4. Displays results in a clean and understandable fashion
+
+
+## User Stories
+
+1. As a user, I want a way to automatically save information about flights when searching the web, so that I can easily revisit them in the future.
+
+2. As a user, I want to be able to delete specific flights in my history, so that I can hide them or clean up my interface.
+
+3. As a user, I want to be able to filter by price, so that I can easily find all the flights within my price range.
+
+4. As a user, I want to store the relevant information about my flight such as price, airports, time, etc., so that I can get an overview of a flight.
+
+5. As a user, I would like to add additional settings to avoid adding specific flights to my history, so that my history only involves what I am interested in.
+
+6. As a frequent traveler, I want to filter my flight history by destination, so that I can quickly find past searches for trips to my favorite locations.
+
+7. As someone with specific luggage needs, I want to filter my flight history based on luggage allowances, so that I can find the most accommodating airlines without re-entering search criteria.
+
+8. As a traveler with flexible dates, I want the option to sort my flight history by date ranges, so that I can compare prices across different periods and find the cheapest options.
+
+9. As a user who values efficiency, I want the ability to quickly access detailed information on any flight in my history, so that I can make informed decisions without re-searching the flight.
+
+10. As a user who often compares flights, I want the ability to compare selected flights from my history side-by-side, so that I can easily see differences in price, timing, and other relevant details
+ 
+11. As a user with a preference for certain seats, I want to see information on seat availability and type (window, aisle, extra legroom) for flights in my history, so I can make bookings based on my seating preferences.
+
+12. As a user who often compares flights, I want the ability to compare selected flights from my history side-by-side, so that I can easily see differences in price, timing, and other relevant details.
+
+13. As a luxury traveler, I want to filter my flight history to show only business or first-class options, so I can compare premium travel experiences across different airlines.
+
+14. As a parent traveling, I want to filter and sort my flight history based on family-friendly features, such as in-flight entertainment and child meal options, so I can choose the best flights for family travel.
+
+15. As a pet owner, I want to filter my flight history by pet-friendly flights 
+
+## Notes
+
+Potential challenges:
+1. Detecting when to extract data
+2. How will we extract the data from Google Flights.
+
+Features to highlight:
+1. Automatically store information about the flights you’re looking at
```

Comment:
This would probably get clearer as you develop your product, but I'm curious about how exactly flights would be added to user's flight lists. Would you add all the flights that appear in the google search or maybe just the flights that they click on?